addons/mil_logistics/fnc_ML.sqf

```diff
@@ -1164,16 +1164,17 @@ switch(_operation) do {
 
                     _eventID = [_event, "id"] call ALIVE_fnc_hashGet;
                     _eventData = [_event, "data"] call ALIVE_fnc_hashGet;
-                    
-                    //Sanitize _eventForceMakeup, 0 is the minimum for every reinforcement type
-                    _eventForceMakeup = (_eventData select 3) apply { _x max 0 };
-                    
+                    _eventForceMakeup = _eventData select 3;
```

Comment:
Maybe instead of doing this here at all, we could move the _eventForceMakeup call into the if statements? On line 1237, we could move the _eventForceMakeup = _eventData select 3 to there. That way we have a very obvious if/else for setting that data instead of how it is currently, where we set the data at the start before the if/else statement and then modify it in there.

Not a huge deal, just a minor nitpick to try and make it as obvious as possible what's happening. Looks good otherwise, should fix the issue atm. Larger logi revamp should hopefully (knock on wood) render this less of a problem.

Reply:
I will merge this for now. If the proposed alteration is performed this merge will be reverted and replaced.